### PR TITLE
sys-boot/uefi-mkconfig: new package, add 1.4

### PR DESCRIPTION
**Pull request by Biosias**
Add ebuild for [uefi-mkconfig](https://github.com/Biosias/uefi-mkconfig) script for managing UEFI entries, to be used with sys-kernel/installkernel

@AndrewAmmerlaan

### CODE REVIEW

File: sys-boot/uefi-mkconfig/uefi-mkconfig-1.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	einstalldocs
+}
+
+pkg_postinst() {
+	elog "uefi-mkconfig: Automatic management of UEFI entries"
+	elog "Run uefi-mkconfig while having all efi partitions mounted"
+	elog "Please use with care, this package was tested on a limited number of machines"
+	elog "Some problems may arise due to different implementations of UEFI"
+	elog ""
+	optfeature "Regenerating UEFI configuration on each kernel installation" \ "sys-kernel/installkernel[-systemd,efistub]"
```
Comment (AndrewAmmerlaan) [inline sys-boot/uefi-mkconfig/uefi-mkconfig-1.4.ebuild:34]:
```suggestion
	elog 
	optfeature "Regenerating UEFI configuration on each kernel installation" \ 
	     "sys-kernel/installkernel[-systemd,efistub]"
```

We don't need to give an empty string argument to elog to print an empty line. And the optfeature line we can make a bit shorter, so pkgcheck does not complain.

**Review by AndrewAmmerlaan — COMMENTED** [review]
Could you squash all commits together and rebase? We like to have single atomic commits, and for `::gentoo` we use rebasing instead of merge commits.

**Comment by AndrewAmmerlaan** [comment]
> ## Missing Bugzilla accounts
> 
> **WARNING**: The following maintainers do not match any Bugzilla accounts:
> 
>     * [jakub@gdos.sk](mailto:jakub@gdos.sk)
> 
> 
> Please either fix the e-mail addresses in metadata.xml or create a Bugzilla account, and request reassignment afterwards.

And please also register this e-mail address on bugs.gentoo.org, or use an e-mail address here that matches your bugs.gentoo.org account. Otherwise you'll not receive bug notifications from users or tinderbox.

**Comment by Biosias** [comment]
> > ## Missing Bugzilla accounts
> > **WARNING**: The following maintainers do not match any Bugzilla accounts:
> > ```
> > * [jakub@gdos.sk](mailto:jakub@gdos.sk)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Please either fix the e-mail addresses in metadata.xml or create a Bugzilla account, and request reassignment afterwards.
> 
> And please also register this e-mail address on bugs.gentoo.org, or use an e-mail address here that matches your bugs.gentoo.org account. Otherwise you'll not receive bug notifications from users or tinderbox.

I've just created bugzilla account with that email

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**